sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -1076,7 +1076,9 @@ case class ShowCreateTableCommand(table: TableIdentifier)
             "Failed to execute SHOW CREATE TABLE against table " +
               s"${tableMetadata.identifier}, which is created by Hive and uses the " +
               "following unsupported feature(s)\n" +
-              tableMetadata.unsupportedFeatures.map(" - " + _).mkString("\n")
+              tableMetadata.unsupportedFeatures.map(" - " + _).mkString("\n") + ". " +
+              s"Please use `SHOW CREATE TABLE ${tableMetadata.identifier} AS SERDE` " +
+              "to show Hive DDL instead."
```

Comment:
Do we have any test for this error case?

Reply:
Yea, test case `hive partitioned view is not supported` in `HiveShowCreateTableSuite` is for this error.

Reply:
Oh, I see. Thanks.